genref/config.yaml

```diff
@@ -0,0 +1,147 @@
+
+hideTypePatterns:
+  - "ParseError$"
+  # - "List$"
+
+markdownDisabled: false
+
+stripPrefix:
+  - k8s.io/api/
+  - k8s.io/apimachinery/pkg/apis/
+
+apis:
+  - name: kubelet-config
+    package: k8s.io/kubelet
+    path: config/v1beta1
+    includes:
+      - k8s.io/component-base/config/v1alpha1
+
+  - name: kubelet-stats
+    package: k8s.io/kubelet
+    path: pkg/apis/stats/v1alpha1
+    # This is skipped because the 'stats' API is not yet available in 0.19.3
+    skip: true
+
+  - name: kube-scheduler-config
+    package: k8s.io/kube-scheduler
+    path: config/v1
+
+  - name: kube-scheduler-config
+    package: k8s.io/kube-scheduler
+    path: config/v1beta1
+
+  - name: kube-controller-manager-config
+    package: k8s.io/kube-controller-manager
+    path: config/v1alpha1
+
+  # TODO: This is shared between different controllers so it should be
+  # an includes entry rather than a standalone package.
+  - name: controller-manager-common
+    package: k8s.io/controller-manager
+    path: config/v1alpha1
+
+  - name: kube-proxy-config
+    package: k8s.io/kube-proxy
+    path: config/v1alpha1
+
+  - name: metrics
+    package: k8s.io/metrics
+    path: pkg/apis/metrics/v1alpha1
+
+  - name: metrics
+    package: k8s.io/metrics
+    path: pkg/apis/metrics/v1beta1
+
+  - name: clientauthentication
+    package: k8s.io/client-go
+    path: pkg/apis/clientauthentication/v1alpha1
+
+  - name: clientauthentication
+    package: k8s.io/client-go
+    path: pkg/apis/clientauthentication/v1beta1
+
+  - name: apiserver-config
+    package: k8s.io/apiserver
+    path: pkg/apis/apiserver/v1alpha1
+
+  # This only exists in v0.19.3
+  - name: apiserver-config
+    package: k8s.io/apiserver
+    path: pkg/apis/apiserver/v1beta1
+
+  - name: apiserver-config
+    package: k8s.io/apiserver
+    path: pkg/apis/apiserver/v1
+
+  - name: apiserver-encryption
+    package: k8s.io/apiserver
+    path: pkg/apis/config/v1
+
+  - name: apiserver-audit
+    package: k8s.io/apiserver
+    path: pkg/apis/audit/v1
+
+  - name: apiserver-webhookadmission
+    package: k8s.io/apiserver
+    path: pkg/admission/plugin/webhook/config/apis/webhookadmission/v1alpha1
+
+  - name: apiserver-webhookadmission
+    package: k8s.io/apiserver
+    path: pkg/admission/plugin/webhook/config/apis/webhookadmission/v1
+
+  - name: apiserver-eventratelimit
+    package: k8s.io/kubernetes
+    path: plugin/pkg/admission/eventratelimit/apis/eventratelimit/v1alpha1
+    # Skipped because k8s.io/kubernetes cannot be imported
+    skip: true
+
+  - name: apiserver-podtolerationrestriction
+    package: k8s.io/kubernetes
+    path: plugin/pkg/admission/podtolerationrestriction/apis/podtolerationrestriction/v1alpha1
+    # Skipped because k8s.io/kubernetes cannot be imported
+    skip: true
+
+  # This is for v0.19.3
+  - name: cloud-provider-config
+    package: k8s.io/cloud-provider
+    path: app/apis/config/v1alpha1
+    skip: true
+
+  # This is for v0.20.x
```

Comment:
What does line 124 mean?
`v0.20.x`?

Reply:
the config between 119-112 and the one between 125-131 are both for cloud-provider. If we are generating docs for  the package version v0.19.3, we need to change line 122 to `skip: false`. If however we are generating docs for v0.20.x, we need to use line 125-131. This is because the package path has been changed upstream.